pkg/preflight/execute.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // execute will execute the preflights using spec in preflightSpec.
 // This spec should be rendered, no template functions remaining
-func execute(appID string, sequence int64, preflightSpec *troubleshootv1beta2.Preflight, ignorePermissionErrors bool) (*troubleshootpreflight.UploadPreflightResults, error) {
+func Execute(appID string, sequence int64, preflightSpec *troubleshootv1beta2.Preflight, ignorePermissionErrors bool, isHeadless bool) (*troubleshootpreflight.UploadPreflightResults, error) {
```

Comment:
I don't think this function should be used here, so there should be no need for `isHeadless` flag.  When preflights are running from CLI, the entire thing should just be delegated to the preflight package in the troubleshoot project.  CLI only needs to template the specs out.  Probably best way to do this is to pass templating context to troubleshoot.  I'm open to suggestions here though.

Reply:
ya definitely makes sense, thanks